cli2/cli.ts

```diff
@@ -46,7 +46,8 @@ async function make(
   const dependencyConfig = DependencyConfig.parse({
     dependencies: morphirJson.dependencies,
     localDependencies: morphirJson.localDependencies,
-    includes: includes
+    includes: includes, 
+    projectDir: projectDir
   })
```

Comment:
ProjectDir preserves the project directory option passed through the command line